docs/pr-review-guidelines.md

```diff
@@ -0,0 +1,30 @@
+# Postman PR review guidelines
+
+This document sets baseline standard for our PR reviews and serves as a mechanism to align individual postmangineer's expectations of a PR review, both from a reviewer's and am author's point of view.
+
+## Principles
+
+- Technical facts and data overrule opinions and personal preferences.
+- Decisions should **always** been made through either solid engineering principles, and/or through empirical data. If there're multiple equally valid alternatives, reviewers should accept the author's preference.
+- Any purely style points (i.e. whose no valid argument via engineering principles nor data) should be consistent with whatever that's in the codebase. If there's no previous style, accept the author's
```

Comment:
My own view is:
- Current codebase has pretty poor style
- I wonder if this merely shifts the debate back one level, i.e. what do we consider style vs engineering principles/data

For example, current codebase has some serious nested if statements going on, which I think is quite bad, but I'm not sure if that's style or engineering principles